### PR TITLE
Mckenzieja Pulling a Request

### PR DESCRIPTION
**Pull request by mckenzieja**

### CODE REVIEW

File: scripts/hello_world_mckenzieja.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function sup(){
```
Comment (StevenCopeland) [inline scripts/hello_world_mckenzieja.js]: Where are you comments with the requirements??? See issue #1292

Comment (mckenzieja) [reply scripts/hello_world_mckenzieja.js]: Updated!

Comment (StevenCopeland) [reply scripts/hello_world_mckenzieja.js]: Great! Looks good now!

**Comment by Roshanjossey** [comment]
Hi  @mckenzieja, I'm working on a similar project to help beginners to get started with contributing to open source projects.

Do check it out https://github.com/Roshanjossey/first-contributions

**Pull request merged**